src/altinn-app-frontend/src/features/instantiate/containers/InstantiationContainer.tsx

```diff
@@ -41,7 +41,9 @@ function InstantiateContainer(props: IInstantiateContainerProps) {
         profile={profile}
         type={props.type}
       />
-      {children}
+      <main id='main-content'>
```

Comment:
Maybe add `role="main"` to the main-container for extended assistive support.

Reply:
No need:
`The [<main>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) element has a role of main. Developers should use semantic HTML -- in this case <main> -- over using ARIA.`

Reply:
Indeed! Maybe this is added automatically by the compiler if we support lesser browser versions.